docroot/modules/custom/layout_builder_custom/src/EventSubscriber/SectionComponentSubscriber.php

```diff
@@ -88,46 +90,91 @@ public function onBuildRender(SectionComponentBuildRenderArrayEvent $event) {
     if (isset($build['#plugin_id'])) {
       switch ($build['#plugin_id']) {
         case 'inline_block:uiowa_card':
-        case 'inline_block:uiowa_image':
-          if (isset($build['#attributes']['class'])) {
-            if ($build['#plugin_id'] === 'inline_block:uiowa_card') {
-              // Map the layout builder styles to the view mode to be used.
-              $media_formats = [
-                'media--circle' => 'large__square',
-                'media--square' => 'large__square',
-                'media--ultrawide' => 'large__ultrawide',
-                'media--widescreen' => 'large__widescreen',
-              ];
+          // Use the card render element.
+          $build['#type'] = 'card';
+          unset($build['#theme']);
+
+          $build['#attributes']['class'][] = 'block';
+
+          $content = $build['content'];
+          $mapping = [
+            '#media' => 'field_uiowa_card_image',
+            '#subtitle' => 'field_uiowa_card_author',
+          ];
+
+          // Map fields to the card parts.
+          foreach ($mapping as $prop => $fields) {
+            if (!is_array($fields)) {
```

Comment:
not sure i understand this check unless someone messes with $mapping up above.

Reply:
Originally, I was working with idea that the array values in `$mapping` could be an array because sometime we add multiple items to a single slot (e.g. meta) on the card.